cli/chronos.py

```diff
@@ -75,3 +76,23 @@ def getCurrentImageVersion(self, roger_env, environment, application):
 
     def getTasks(self, roger_env, environment):
         print("Not yet implemented!")
+
+    def image_name(
```

Comment:
I think we should then go with the pep-008 standards, just need to be consistent with function names then